tools/rails.md

```diff
@@ -0,0 +1,31 @@
+---
+permalink: /rails
+layout: post
+title: Rails
+link: https://rubyonrails.org/
+releaseDateColumn: true
+category: framework
+activeSupportColumn: true
```

Comment:
Set this to false, since the `support` field is not set in the releases.

Reply:
Done, though `support` is set on all releases except the latest. I don't think there's a way to show this, but v5.2 is in active support (bug fixes as well as security fixes) but with no firm end date (the end date is "whenever 6.0 is finally released").

Reply:
https://deploy-preview-26--endoflife.netlify.com/rails shows "NA" for all the releases, including the ones that are green. I marked them as `eol: false`. What did I forget?

Reply:
None of the tools (so far) with custom EOL dates (eol: true/false) had the release column shown.

This will need some fixes in the layout. I'll take it up :soon:

Reply:
@jmarianer Since all versions don't have an `eol`. You can hide the release column by setting `releaseColumn` to false

Reply:
I do what the release column to show, though. I decided to look into the layout change @captn3m0 mentioned and added it to this PR. (I also changed the hash to an array as required by a recent commit).